workflow/tests/ACCA_Examples/Smith_Residence.xml

```diff
@@ -61,6 +61,72 @@
           </extension>
         </WeatherStation>
       </ClimateandRiskZones>
+      <Zones>
+        <Zone>
+          <SystemIdentifier id='SmithResidenceConditioned'/>
+          <ZoneType>conditioned</ZoneType>
+          <Spaces>
+            <Space>
+              <SystemIdentifier id='bedroom_1'/>
+              <FloorArea>238.0</FloorArea>
+            </Space>
+            <Space>
+              <SystemIdentifier id='bedroom_2'/>
+              <FloorArea>210.0</FloorArea>
+            </Space>
+            <Space>
+              <SystemIdentifier id='bedroom_3'/>
+              <FloorArea>224.0</FloorArea>
+            </Space>
+            <Space>
+              <SystemIdentifier id='hall_closets'/>
+              <FloorArea>126.0</FloorArea>
+            </Space>
+            <Space>
+              <SystemIdentifier id='bathroom_1'/>
+              <FloorArea>63.0</FloorArea>
+            </Space>
+            <Space>
+              <SystemIdentifier id='bathroom_2'/>
+              <FloorArea>35.0</FloorArea>
+            </Space>
+            <Space>
+              <SystemIdentifier id='entry'/>
+              <FloorArea>98.0</FloorArea>
+            </Space>
+            <Space>
+              <SystemIdentifier id='living_dining'/>
+              <FloorArea>504.0</FloorArea>
+            </Space>
+            <Space>
+              <SystemIdentifier id='kitchen_stairwell'/>
+              <FloorArea>294.0</FloorArea>
+            </Space>
+            <Space>
+              <SystemIdentifier id='workshop'/>
+              <FloorArea>340.0</FloorArea>
+            </Space>
+            <Space>
+              <SystemIdentifier id='recroom'/>
+              <FloorArea>512.0</FloorArea>
+            </Space>
+            <Space>
+              <SystemIdentifier id='laundry_equipment'/>
+              <FloorArea>204.0</FloorArea>
+            </Space>
+          </Spaces>
+        </Zone>
+        <Zone>
+          <SystemIdentifier id='SmithResidenceCrawlSpace'/>
+          <ZoneType>unconditioned</ZoneType>
+          <Spaces>
+            <Space>
+              <SystemIdentifier id='crawlspace'/>
+              <FloorArea>736.0</FloorArea>
+            </Space>
+          </Spaces>
+        </Zone>
```

Comment:
Why are we bothering to add unconditioned zones/spaces? I suggest removing from any of these files.

Reply:
Oh it's created when I saw some surfaces that are interior adjacent to crawlspace. I hesitated when I was doing so, I'm fine removing those.